Headers/DebugServer2/Utils/Enums.h

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2014-present, Facebook, Inc.
+// All rights reserved.
+//
+// This source code is licensed under the University of Illinois/NCSA Open
+// Source License found in the LICENSE file in the root directory of this
+// source tree. An additional grant of patent rights can be found in the
+// PATENTS file in the same directory.
+//
+
+template <typename Enum> struct EnableBitMaskOperators {
```

Comment:
You should wrap all if this in 
```
namespace ds2 {
namespace Utils {
}
}
```

Reply:
AFIAK you can't. Similar issue as above. You'd run into problems with the operators being defined in a different namespace than the type they are defined for. `ds2::utils::operator|` generic over `ds::BreakpointManager::Lifetime`.